src/service.ts

```diff
@@ -293,31 +298,30 @@ export class Service {
     return JSON.parse(await response.text());
   }
 
-  static saveJSON(json: object, uri: string): Promise<string> {
+  static saveJSON(json: object, uri: string): any {
```

Comment:
before the change this method returned url, now it is not. try to make method async?